test-integration/scores.coffee

```diff
@@ -30,6 +30,10 @@ describe 'HS Student Scores', ->
     @scores.el.periodTab().click()
     @user.goToHome()
 
+  @it 'generates export', ->
+    @scores.el.generateExport().click()
+    @scores.getExportDownloadPath().then (downloadedFilePath) =>
```

Comment:
`get*` seems a little misleading (I would think that method would return a path, not `yield` a path). Maybe `.waitForDownloadPath` since it waits first?